packages/cells/src/model.ts

```diff
@@ -164,7 +164,7 @@ export class CellModel extends CodeEditor.Model implements ICellModel {
   constructor(options: CellModel.IOptions) {
     super({ modelDB: options.modelDB });
 
-    this.id = options.id || UUID.uuid4();
+    this.id = options.id || (options.cell?.id as string) || UUID.uuid4();
```

Comment:
Test:
- [ ] If a cell is created without an option id, but does have a cell id, that is the id that is used for the cell